plugin/lion.vim

```diff
@@ -62,6 +62,22 @@ function! s:align(mode, type, vis, align_char)
 		endif
 		let [start_line, end_line, start_col, end_col, middle_start_col, middle_end_col] = pos
 
+		" Check for 'lion_squeeze_spaces' option
+		if get(b:, 'lion_squeeze_spaces', 0)
```

Comment:
A likely use case might be to turn on the global option, but disable it for some buffers (perhaps with an auto command for a particular file type). In order to support that, we'd need to distinguish between 0 and unset.